rdmo/projects/tests/test_validator_value_type.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+
+from rest_framework.exceptions import ValidationError as RestFameworkValidationError
+
+from ..validators import ValueTypeValidator
+
+data = (
```

Comment:
I have maybe a couple validation test cases to add:

```py
data = (
...
('boolean', 'true'),
('boolean', 'false'),

('email', 'user.name@example.com'),
('email', 'user.name+tag@example.com'),
...
)

data_error = (
...
('boolean', 'truee'),
('boolean', 'falze'),

('email', 'user@com'),
('email', 'user@.com'),

('date', '01.13.2024'),
('date', '32.01.2024'),
('date', '13/32/2024'),

('datetime', '2024-01-02T25:00'),
('datetime', '2024-01-02T10:60:00'),
...
)

Reply:
I added the cased except the wrong dates, since the validator is not able to catch those. I don't consider this big of a problem. If people want to be sure, they need to use the datepicker.